tests/sentry/api/endpoints/test_user_details.py

```diff
@@ -57,11 +58,11 @@ def test_superuser_simple(self):
         assert "identities" in resp.data
         assert len(resp.data["identities"]) == 0
 
-    @with_feature("auth:enterprise-staff-cookie")
     def test_staff_simple(self):
         self.login_as(user=self.staff_user, staff=True)
 
-        resp = self.get_success_response(self.user.id)
+        with override_options({"staff.user-email-allowlist": [self.staff_user.email]}):
```

Comment:
if you put staff email as a class variable, could you use the `@override_options` decorator?

Reply:
you know that that's pretty smart, I prob should've done that instead of all the `with` stuff

Reply:
Will have to switch everything over sad

Reply:
ok on second lookover, some of the stuff makes sense to use `with` for the ones with very few tests